magento/magento2
# add check if attribute value is possible to be set as configurable option

a none existing value would be set and the dropdown would show an empty value (not the please select message)

### Description (*)

Configurable options of a product can be preselected when adding url-parameters for each option (http://shop.sample.com/product-url#<<option1>>=<<option-value1>>). if the option value does not exist, the value would still be set for the option. the result is than an empty dropdown for this option.

with adaption of this value setting mechanism to additionally check if the option value for the option is allowed the wrongly set value is ignored.

### Manual testing scenarios (*)

1. go to a product detail page of a configurable product
2. get the option id and one option value from the html
3. add the option id and value to the url (with ...#<<optionid>>=<<optionvalue>>
4. with an existing option value the according option value should be shown in the product detail page
5. with a non existing option value the according option should just show the please select option

### Contribution checklist (*)
